### PR TITLE
Use skyhook for searching

### PR DESCRIPTION
**Pull request by markus101**
@Taloth @kayone

### CODE REVIEW

File: src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs [hunk 1]
```diff
@@ -10,23 +13,29 @@
...
+
+        public SkyHookProxy(IHttpClient httpClient, Logger logger)
         {
-            _logger = logger;
             _httpClient = httpClient;
+            _logger = logger;
 
-            _requestBuilder = new HttpRequestBuilder("http://skyhook.sonarr.tv/v1/tvdb/shows/en/");
+//            _requestBuilder = new HttpRequestBuilder("http://skyhook.sonarr.tv/v1/tvdb/{route}/en/");
+            _requestBuilder = new HttpRequestBuilder("http://localhost:2000/v1/tvdb/{route}/en/");
```
Comment (Taloth) [inline src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs]: DEBUG CODE

File: src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs [hunk 2]
```diff
@@ -186,5 +247,22 @@ private static MediaCoverTypes MapCoverType(string coverType)
             }
         }
 
+        private static string GetSearchTerm(string phrase)
+        {
+            phrase = phrase.RemoveAccent();
```
Comment (Taloth) [inline src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs]: Maybe we should do this on the server.

**Comment by Taloth** [comment]
Get rid of the tvdbproxy assembly too.

File: src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs (hunk 2 above)
Comment (markus101) [inline src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs]: I was thinking the same, its very TVDB specific. I will move it over.

**Pull request merged**